libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -10,9 +13,21 @@ import { getReadingList, removeFromReadingList } from '@tmo/books/data-access';
 export class ReadingListComponent {
   readingList$ = this.store.select(getReadingList);
 
-  constructor(private readonly store: Store) {}
+  constructor(
+    private readonly store: Store,
+    private snackBar: MatSnackBar
+  ) {}
 
   removeFromReadingList(item) {
     this.store.dispatch(removeFromReadingList({ item }));
+    const sbInstance = this.snackBar.open(
```

Comment:
Same comment as before. Shouldn't the snackbar appear after you have confirmed the success of `removeFromReadingList` action (observing `confirmedRemoveFromReadingList`)?

Reply:
@mandarini: Review addressed, please take a look.